lib/ruby_llm/chat.rb

```diff
@@ -13,10 +13,19 @@ class Chat
 
     attr_reader :model, :messages, :tools
 
-    def initialize(model: nil)
+    def initialize(model: nil) # rubocop:disable Metrics/MethodLength
       model_id = model || RubyLLM.config.default_model
       self.model = model_id
-      @temperature = @model.metadata['family'] == 'o1' ? 1 : 0.7
```

Comment:
I don't think this was ever working, the `family` is one level above the `metadata`.